src/main/java/de/hysky/skyblocker/skyblock/garden/visitor/VisitorHelper.java

```diff
@@ -154,8 +161,13 @@ private static ItemStack getCachedItem(String itemName) {
 	/**
 	 * Draws the visitor items and their associated information.
 	 */
-	private static void renderVisitorHelper(DrawContext context, TextRenderer textRenderer) {
+	public void renderWidget(DrawContext context, int mouseX, int mouseY, float deltaTicks) {
+		TextRenderer textRenderer = MinecraftClient.getInstance().textRenderer;
 		int index = 0;
+		int newWidth = 0;
+		int x = getX() + PADDING;
+		int y = getY() - (int) (textRenderer.fontHeight / 2f - ICON_SIZE * 0.95f / 2) + PADDING;
+		context.fill(getX(), getY(), getRight(), getBottom(), 0x28_00_00_00);
```

Comment:
This seems a little too dark and almost entirely blends in with the darkened background of the open screen. Would something like `0x10FFFFFF` be good, or is that now too bright?

`0x10FFFFFF`:
<img width="477" height="289" alt="image" src="https://github.com/user-attachments/assets/5d02a0f5-54af-400d-8694-520fd33f3905" />

Reply:
For reference, this is how it is now (`0x28000000`):
<img width="456" height="143" alt="image" src="https://github.com/user-attachments/assets/f1daddb0-50d9-47d8-a845-d89a2126b8ea" />